dump/create_pel.cpp

```diff
@@ -129,6 +130,50 @@ uint32_t createSbeErrorPEL(const std::string& event, const sbeError_t& sbeError,
     return plid;
 }
 
+uint32_t createPOZSbeErrorPEL(const std::string& event,
+                              const sbeError_t& sbeError,
+                              const FFDCData& ffdcData)
+{
+    uint32_t plid = 0;
+    auto ffdcList = sbeError.getFfdcFileList();
+
+    // poz sbe errors are created only when there is FFDC data
+    for (auto& iter : ffdcList)
+    {
+        FFDCInfo pelFFDCInfo;
+        log<level::INFO>(
+            std::format("createPOZSbeErrorPEL capturing FFDC data for",
+                        "SLID={}", iter.first)
+                .c_str());
+        auto tuple = iter.second;
+        uint8_t severity = std::get<0>(tuple);
+
+        // convert fapi error to pel error
+        Level logSeverity = Level::Error;
+        if (severity == static_cast<uint8_t>(
+                            openpower::phal::FAPI2_ERRL_SEV_UNDEFINED) ||
+            severity ==
+                static_cast<uint8_t>(openpower::phal::FAPI2_ERRL_SEV_RECOVERED))
+        {
+            logSeverity = Level::Informational;
+        }
+        else if (severity == static_cast<uint8_t>(
+                                 openpower::phal::FAPI2_ERRL_SEV_PREDICTIVE))
+        {
+            logSeverity = Level::Warning;
+        }
+        pelFFDCInfo.emplace_back(
+            std::make_tuple(sdbusplus::xyz::openbmc_project::Logging::server::
+                                Create::FFDCFormat::Custom,
+                            static_cast<uint8_t>(0xCB),
```

Comment:
Better to define these HEX values as `constexpr someVariableName` for better clarity and maintainability